lib/engine/game/g_1825/meta.rb

```diff
@@ -119,6 +119,118 @@ module Meta
             desc: 'Do not increase bank size based on number of minors and kits',
           },
         ].freeze
+
+        MUTEX_RULES = [
```

Comment:
exclusive_rules, standalone rules?

Reply:
sorry, MUTEX is a verilog term for "mutually exclusive'